libc/src/__support/FPUtil/dyadic_float.h

```diff
@@ -101,6 +101,7 @@ template <size_t Bits> struct DyadicFloat {
     return exponent + (Bits - 1);
   }
 
+#ifdef LIBC_TYPES_HAS_FLOAT16
```

Comment:
Why does this require float16?

Reply:
Because of this line: https://github.com/llvm/llvm-project/blob/3db0f8c895d4e814a18b754f9afbb1e03bd839a5/libc/src/__support/FPUtil/dyadic_float.h#L108